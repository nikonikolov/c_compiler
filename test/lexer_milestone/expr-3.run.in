
int ff()
{

int x=0 & 0;
int u=x | x;
int y=u^~x;


 return y;
}
