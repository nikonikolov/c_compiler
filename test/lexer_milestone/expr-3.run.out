int Keyword Tint 2  2
ff Identifier TIdentifier 2  2
( Operator T( 2  2
) Operator T) 2  2
{ Operator T{ 3  3
int Keyword Tint 5  5
x Identifier TIdentifier 5  5
= Operator T= 5  5
0 Constant TIntConstant 5  5
& Operator T& 5  5
0 Constant TIntConstant 5  5
; Operator T; 5  5
int Keyword Tint 6  6
u Identifier TIdentifier 6  6
= Operator T= 6  6
x Identifier TIdentifier 6  6
| Operator T| 6  6
x Identifier TIdentifier 6  6
; Operator T; 6  6
int Keyword Tint 7  7
y Identifier TIdentifier 7  7
= Operator T= 7  7
u Identifier TIdentifier 7  7
^ Operator T^ 7  7
~ Operator T~ 7  7
x Identifier TIdentifier 7  7
; Operator T; 7  7
return Keyword Treturn 10  10
y Identifier TIdentifier 10  10
; Operator T; 10  10
} Operator T} 11  11
